airbyte-integrations/connectors/source-pingdom/source_pingdom/schemas/checks.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
Missing additionalProperties = true. @KimPlv can you run `airbyte-ci format fix all`?

Reply:
done :white_check_mark: